src/storageDataGroup.js

```diff
@@ -337,7 +336,6 @@ class StorageDataGroup {
                         acc[curr.name] = curr;
                         return acc;
                     }, {});
-                    return Promise.resolve();
```

Comment:
This is quite the change in behavior for users that are relying on minimizing writes to the data group on the BIG-IP. I just want to make sure that we don't catch other users off guard with this change if they're trying to minimize writes to the data-group. We might need to include an option to return before the write if users want to maintain this behavior. @mstokes-f5, what are your thoughts?

Reply:
FYI, Mitchell and I chatted about this while I was doing the change. We talked about making this optional, but he felt that he'd rather try it on FAST first and see if it is an issue this way.

Reply:
Oh, OK, if you two already talked then I think that covers the majority of projects that will be affected by this. We can always add the option in at a later time if someone requests it. Thanks